### DISCUSSION TITLE
document deploy bunjs based app

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [oven-sh/bun#6134] [General]
I know there are several options to deploy bunjs based apps, and probably bun team may create his own plateforme for deploying bun based apps like deno deploy. that said I hope in bun doc that document how to deploy bun apps ? 
for learning purpose it is important how deploy bun using domcloud.co  , domcloud have some similaties with heroku but with free plan

**Comment by ani-ali** [comment]
for free deployment i tried so many but didn't found such a thing.
you can use render if you want but it's not good to be honest on free plan.

use hono for backend, similar + fast performance + deploy freely on vercel with unlimited projects
